organizations/middleware.py

```diff
@@ -37,7 +37,8 @@ def get_schema(self, request):
         tenant = self.get_tenant(request)
         if tenant:
             return tenant.schema_name
-        return None
+        # as get_schema is being used when querying BigQuery datasets, we explicity need to mention `public`
+        return "public"
```

Comment:
I think this could be covered by tests easily right?

Reply:
Yeah, this is already being covered as any API calls will be running this middleware.

The only thing that wasn't running was the statements when we have tenant present. They should now also run in one test case.